functional-test/src/test/groovy/org/rundeck/util/container/BaseContainer.groovy

```diff
@@ -350,36 +350,40 @@ abstract class BaseContainer extends Specification implements ClientProvider {
     }
 
     /**
-     * Executes a job identified by jobId and waits until the job execution is completed.
-     *
-     * @param jobId The identifier of the job to run.
-     * @param body Additional parameters for the job execution. Default is null.
-     * @return A Map containing the final execution details.
+     *  Use the similar method in JobUtils
+     * @param jobId
+     * @param body
+     * @return
      */
+    @Deprecated
```

Comment:
you should add doc indicating what method to use instead

Reply:
There is no 1 to 1 method that can be used in its place.
The method's javadoc says `Use the similar method in JobUtils`.
I've removed usage of this method from `rundeck`. 
I will get to `pro` in the next ticket and, hopefully, would be able to delete it entirely.